src/main/resources/Macros/MapMacro/Code/Configuration.xml

```diff
@@ -62,6 +62,7 @@
       <apiKey>
         <customDisplay/>
         <disabled>0</disabled>
+        <hint/>
```

Comment:
This is a new property introduced in https://jira.xwiki.org/browse/XWIKI-7783 (9.11-rc1) and it will cause issues at install on old versions: https://jira.xwiki.org/browse/XWIKI-15028. It looks like you worked on a new XWiki version, which is the recommended way, but we need to get rid of this property, either manually or working on the minimal XWiki version supported by this macro.

Reply:
ok Alex, thanks for the info, I removed the hint